### DISCUSSION TITLE
Flooded with alerts for webex

### DISCUSSION THREAD

**Discussion by pavlos1982** [Security-Onion-Solutions/securityonion#5488] [General] [answered]
I have some users running video conferencing on Webex, and my security onion is flooded with the following alerts: **ET INFO Session Traversal Utilities for NAT (STUN Binding Request On Non-Standard High Port)**

is there a way to ignore these alerts? As I have checked the IP it belongs to Webex servers, and I doubt there is anything malicious with this, and the user is running Webex for online conferencing?

Is it possible to ignore these alerts specifically for Webex or any recommendations?

![webex](https://user-images.githubusercontent.com/5363713/133077378-889b59b6-b0fa-4126-9c3a-6553a37761e1.JPG)

**Comment by dougburks** [comment, marked as answer]
Have you checked the documentation?
https://docs.securityonion.net/en/2.3/managing-alerts.html

**Reply by pavlos1982** [reply]
I have check the document, but how do I identify the SID 2100498, in my case the **ET INFO Session Traversal Utilities for NAT (STUN Binding Request On Non-Standard High Port)**

and I do not want to disable the rule completely only for the Webex application.

**Reply by cac0ns3c** [reply]
You could set suppression for this rule by SID for a specific IP address or subnets

**Reply by pavlos1982** [reply]
how do I do that? thanks

**Reply by cac0ns3c** [reply]
Follow the docs: https://docs.securityonion.net/en/2.3/managing-alerts.html#suppressions
